src/instructlab/model/serve.py

```diff
@@ -56,6 +56,16 @@
     required=False,
     help="Log file path to write server logs to.",
 )
+@click.option(
+    "--chat-template",
+    type=str,
+    help=(
+        "Which chat template to use in serving the model, either 'auto', "
+        "'tokenizer', or a path to template file. \n"
```

Comment:
```suggestion
        "'tokenizer', or a path to jinja formatted template file. \n"
```

It relies on jinja right?